### DISCUSSION TITLE
pandoc: Cannot decode byte '\xcf': Data.Text.Internal.Encoding.decodeUtf8: Invalid UTF-8 stream

### DISCUSSION THREAD

**Discussion by an-pipi** [quarto-dev/quarto-cli#5198] [Q&A]
Hello, everyone:
When I compile in Rstudio, I just use the official quarto example, it has the following problem, but the encoding I save is indeed UTF-8, I want to know how to solve this problem。
`pandoc: Cannot decode byte '\xcf': Data.Text.Internal.Encoding.decodeUtf8: Invalid UTF-8 stream`

**Comment by mcanouil** [comment]
Could you share a reproducible example and/or precise steps to reproduce?
And could you also provide your operating system, version, RStudio version, Quarto version, and other information that could help us help you.

**Reply by an-pipi** [reply]
The code and running results are like this
```
---
title: "ggplot2 demo"
author: "Norah Jones"
date: "5/22/2021"
format: 
  html:
    fig-width: 8
    fig-height: 4
    code-fold: true
---

## Air Quality

@fig-airquality further explores the impact of temperature on ozone level.

```{r}
#| label: fig-airquality
#| fig-cap: "Temperature and ozone level."
#| warning: false

library(ggplot2)

ggplot(airquality, aes(Temp, Ozone)) + 
  geom_point() + 
  geom_smooth(method = "loess"
)
```
and the result is this
```
processing file: 123.qmd
                                                                                                           
output file: 123.knit.md

pandoc 
  to: html
  output-file: 123.html
  standalone: true
  section-divs: true
  html-math-method: mathjax
  wrap: none
  default-image-extension: png
  
metadata
  document-css: false
  link-citations: true
  date-format: long
  lang: en
  
pandoc: Cannot decode byte '\xcf': Data.Text.Internal.Encoding.decodeUtf8: Invalid UTF-8 stream
```
My operating system is windows10, R version is 4.2.3, Rstudio version is RStudio 2023.03.0+386, Quarto is the built-in version in Rstudio, thanks

**Reply by dragonstyle** [reply]
I'm not sure what the cause of this could be, though it is likely something particular to your configuration that isn't present in our local configuration. One guess is the code page being used by Windows. Could try running `chcp` from the powershell/cmd terminal and let us know what code page you're using?

**Reply by cderv** [reply]
Can't reproduce this on Windows FWIW. Can you check that the source file is UTF-8 in RStudio IDE ? I believe it should be the default, but maybe it is isn't. File  > Save with encoding ... 
![image](https://user-images.githubusercontent.com/6791940/232497915-9be85f00-d87e-4b48-b391-107c61e5959f.png)

Also `chcp` in terminal will indeed help us
